tests/ci_before_install.sh

```diff
@@ -0,0 +1,48 @@
+set -x   # Show which command is being run
+
+case $TRAVIS_OS_NAME in
```

Comment:
Since these files have variables that are unique to Travis should we call them `travis_before_install.sh` and `travis_install.sh`?